File: Zapato/app/src/main/java/com/zapato/zapato/Shoe.java
package com.zapato.zapato;

/**
 * Created by agustincards on 3/21/18.
 */

public class Shoe {
   public String name;
   public int size;
   public final int drawableId;

   public Shoe(String n, int s, int drawId){
      name=n;
      size=s;
      drawableId=drawId;
   }
}

File: Zapato/app/src/main/java/com/zapato/zapato/Shoe.kt
package com.zapato.zapato

/**
 * Created by adrian on 2/14/18.
 */


// Shoe object to parse DataSnapshot json
class Shoe {
    var name: String? = null
//    var description: String? = null
//    var image: Int? = null


    override fun toString(): String {
        return name.toString()
    }
}